src/pages/Login.jsx

```diff
@@ -18,217 +18,238 @@ function Login() {
   const googleSignUp = () => {};
 
   return (
-    <div style={{ display: 'flex', justifyContent: 'center' }}>
-      <Card
-        style={{
-          width: 400,
-          display: 'inline-block',
-          flexDirection: 'left',
-          boxShadow: '0px 4px 10px rgba(0, 0, 0, 0.1)',
-        }}
-      >
-        <AboundSignature style={{ marginBottom: 22 }} />
-        <div>
-          {state === 'login' && (
-            <>
-              <div style={{ display: 'flex', flexDirection: 'row' }}>
-                <Button type="link">
-                  <p
-                    style={{
-                      color: '#115740',
-                      textDecorationLine: 'underline',
-                      textUnderlineOffset: 10,
-                      textDecorationThickness: 3,
-                    }}
+    <Card
+      style={{
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        margin: '0',
+      }}
+    >
+      <div style={{ display: 'flex', justifyContent: 'center' }}>
+        <Card
+          style={{
+            width: 400,
+            display: 'inline-block',
+            flexDirection: 'left',
+            border: 'none',
+
+            boxShadow: 'none',
+          }}
+        >
+          <AboundSignature style={{ marginBottom: 22 }} />
+          <div>
+            {state === 'login' && (
+              <>
+                <div style={{ display: 'flex', flexDirection: 'row' }}>
+                  <Button type="link">
+                    <p
+                      style={{
+                        color: '#115740',
+                        textDecorationLine: 'underline',
+                        textUnderlineOffset: 10,
+                        textDecorationThickness: 3,
+                      }}
+                    >
+                      Log In
+                    </p>
+                  </Button>
+                  <Button type="link" onClick={() => setState('signup')}>
+                    Sign Up
+                  </Button>
+                </div>
+
+                <Divider style={{ marginTop: 3 }} />
+
+                <Form>
```

Comment:
[nit] The spacing between the icons and the form fields' text is a little smaller than comfortable on my browser, let's up it a bit to match the figma
![image](https://user-images.githubusercontent.com/45449494/178357234-df254c39-28e2-495d-85db-6c1e6b483839.png)
![image](https://user-images.githubusercontent.com/45449494/178357335-4f6b17af-0dc6-419a-b236-1ca83691174f.png)
vs. Figma
![image](https://user-images.githubusercontent.com/45449494/178357383-7dfc64a3-87b0-46de-86c8-1b3ddf987be0.png)
![image](https://user-images.githubusercontent.com/45449494/178357413-80275ee7-4044-4cde-9418-3dad90c13b7a.png)